### ISSUE TITLE
Output both versionFile and manifest

### ISSUE THREAD

**Issue by adrienkohlbecker** [smysnk/gulp-rev-all#70]
Hello,

When I do:
```
stream
      .pipe(revall.manifest({ fileName: 'manifest.json' }))
      .pipe(gulp.dest(DIST_DIR))
      .pipe(revall.versionFile({ fileName: 'version.json' }))
      .pipe(gulp.dest(DIST_DIR));
```

I'm getting only the manifest. It would be nice to be able to output both!

Also, when I do:
```
stream
      .pipe(revall.manifest({ fileName: 'manifest.json' }))
      .pipe(revall.versionFile({ fileName: 'version.json' }))
      .pipe(gulp.dest(DIST_DIR));
```

Nothing gets output

Thanks!

**Comment by frankhock** [comment]
:+1:

**Comment by smysnk** [comment]
Should be fixed in v0.8.0, please give it a try.
